plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/TrinoS3ClientFactory.java

```diff
@@ -1,244 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
```

Comment:
This is a rather big change with quite a lot of impact.

Let's concentrate eventually on this kind of changes on the follow-up PR.